CHANGELOG.rst

```diff
@@ -11,12 +11,14 @@ Unreleased
 - Use symlinks on creating venv's when available (`PR #274`_, Fixes `#271`_)
 - Error sooner if pip upgrade is required and fails (`PR #288`_, Fixes `#256`_)
 - Add a ``runner`` argument to ``ProjectBuilder`` (`PR #290`_, Fixes `#289`_)
+- Better error output (`PR #296`_)
```

Comment:
This could be expanded a bit :thinking:

Reply:
New message: "Hide irrelevant `pep517` error traceback and improve error messages"